generator/src/lib.rs

```diff
@@ -919,11 +919,14 @@ fn generate_function_pointers<'a>(
             let name = &cmd.proto.name;
             let type_name = format_ident!("PFN_{}", name);
 
-            let function_name_c = if let Some(alias_name) = aliases.get(name) {
-                alias_name.to_string()
+            // We might need to generate a function pointer for an extension, where we are given the original
+            // `cmd` and a rename back to the extension alias (typically with vendor suffix) in `rename_commands`:
+            let function_name_c = if let Some(&renamed) = rename_commands.get(name.as_str()) {
```

Comment:
You could use `Option::map_or()` on the `get(...)` return value and get rid of the `if let`.

Reply:
An `unwrap_or()` with `cloned()` ;)